be/src/exprs/agg/factory/aggregate_resolver_minmaxany.cpp

```diff
@@ -79,11 +79,15 @@ struct MaxMinByDispatcherInner {
         if constexpr ((lt_is_aggregate<arg_type> || lt_is_json<arg_type>)&&(lt_is_aggregate<ret_type> ||
                                                                             lt_is_json<ret_type>)) {
             if constexpr (is_max_by) {
-                resolver->add_aggregate_mapping_variadic<arg_type, ret_type, MaxByAggregateData<arg_type>>(
-                        "max_by", true, AggregateFactory::MakeMaxByAggregateFunction<arg_type>());
+                resolver->add_aggregate_mapping_notnull<arg_type, ret_type>(
+                        "max_by", true, AggregateFactory::MakeMaxByAggregateFunction<arg_type, false>());
+                resolver->add_aggregate_mapping_notnull<arg_type, ret_type>(
```

Comment:
Why using `add_aggregate_mapping_notnull`? What will happen if `y` is a nullable column for `max_by(x, y)` ?

Reply:
Are nullable columns handled by `MaxBy` itself now?

Reply:
Oh, it seems that nullable columns are handled by `MaxBy` itself now.

Reply:
yes, null is acceptable output values, so max_by/min_by agg cannot be wrapped by NullableAggFunction